diff --git a/RecoBTag/ImpactParameter/python/impactParameter_cff.py b/RecoBTag/ImpactParameter/python/impactParameter_cff.py
index 5f6a056..dda1fa5 100644
--- a/RecoBTag/ImpactParameter/python/impactParameter_cff.py
+++ b/RecoBTag/ImpactParameter/python/impactParameter_cff.py
@@ -46,3 +46,13 @@ from RecoBTag.ImpactParameter.positiveOnlyJetProbabilityComputer_cfi import *
 from RecoBTag.ImpactParameter.positiveOnlyJetProbabilityJetTags_cfi import *
 
 
+
+# Heiners Exercise with Shrinking Cone
+
+from BTagTutorial.Exercise1.shrinkConeTrackCounting3D2ndComputer_cfi import *
+from BTagTutorial.Exercise1.shrinkConeTrackCountingHighEffBJetTags_cfi import *
+
+from RecoBTag.ImpactParameter.trackCounting3D1stComputer_cfi import *
+from BTagTutorial.Exercise1.trackCounting3D4thComputer_cfi import *
+from BTagTutorial.Exercise1.trackCountingSuperHighEffBJetTags_cfi import *
+from BTagTutorial.Exercise1.trackCountingSuperHighPurBJetTags_cfi import *
